.github/workflows/CI-build-test-publish.yml

```diff
@@ -157,15 +157,15 @@ jobs:
                   "type": "header",
                   "text": {
                     "type": "plain_text",
-                    "text": "🎉 zkVerify Docker Image v${{ github.ref_name }} Released! 🎉"
+                    "text": "🎉 $DOCKER_IMAGE_BUILD_NAME Docker Image v${{ github.ref_name }} Released! 🎉"
```

Comment:
I am not sure if you tested it but you may need the following syntax `${{ env.DOCKER_HUB_ORG }}`. I may be wrong here